scripts/Install-AllSoftware.ps1

```diff
@@ -0,0 +1,23 @@
+# 1. Make sure the Microsoft App Installer is installed:
+#    https://www.microsoft.com/p/app-installer/9nblggh4nns1
+# 2. Run this script as administrator. If running scripts is blocked, you can temporarily unblock them by running 
+#    Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process
```

Comment:
if you can, it would be good to check these two conditions in code and give an appropriate error message.

Reply:
Good idea! My latest push now checks and outputs appropriate error msgs.